mpact/migrations/0012_populate_individuals.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.20 on 2021-05-07 12:10
+
+from django.db import migrations
+
+def populate_individuals(apps, schema_editor):
```

Comment:
this migration seeds the initial `Individual` data model from the current chats.